### PR TITLE
Updating return type of ISwap functions

### PR DESCRIPTION
**Pull request by alicevhenshaw**
## Description

ISwap had an incorrect return type for 2 functions. This is a remnant from when we have expiries.

Quick description:

- [ ] N/A
- [ ] New features
- [X] Bug fixes
- [ ] Typo/small tweaks

### CODE REVIEW

File: source/delegate/package.json [hunk 1]
```diff
@@ -27,7 +27,7 @@
   },
   "dependencies": {
     "@airswap/indexer": "0.4.2",
-    "@airswap/swap": "2.3.2",
+    "@airswap/swap": "2.3.3",
```
Comment (poiresel) [inline source/delegate/package.json]: do we need to increment delegate as its dependency has updated.

File: source/wrapper/package.json [hunk 2]
```diff
@@ -27,7 +27,7 @@
     "solidity-coverage": "^0.6.3"
   },
   "dependencies": {
-    "@airswap/swap": "2.3.2",
+    "@airswap/swap": "2.3.3",
```
Comment (poiresel) [inline source/wrapper/package.json]: increment wrapper as well as its dependency was upgraded.

File: source/delegate/package.json (hunk 1 above)
Comment (alicevhenshaw) [reply source/delegate/package.json]: That's not how hwe've done it in previous PRs. We normally just bump the thing that's changed.
e.g. Ethan's PR this morning https://github.com/airswap/airswap-protocols/pull/319/files

I do see your point though.

Comment (alicevhenshaw) [reply source/delegate/package.json]: Bumped delegate and therefore delegate factory

File: source/wrapper/package.json (hunk 2 above)
Comment (alicevhenshaw) [reply source/wrapper/package.json]: Bumped wrapper

**Pull request merged**